app/femr/ui/controllers/TriageController.java

```diff
@@ -136,6 +138,16 @@ public Result indexPost(int id) {
         IndexViewModelPost viewModel = IndexViewModelForm.bindFromRequest().get();
         CurrentUser currentUser = sessionService.retrieveCurrentUserSession();
 
+        //Alaa Serhan
+        //Get the settings
+        ServiceResponse<SettingItem> settingItemServiceResponse = searchService.retrieveSystemSettings();
+        if (settingItemServiceResponse.hasErrors()) {
+            throw new RuntimeException();
+        }
+
+
+        SettingItem settings = settingItemServiceResponse.getResponseObject();
```

Comment:
I will fix that. It was being user previously. I left it behind when I moved everything to Service Layer.